src/mapml.css

```diff
@@ -23,37 +23,145 @@
 .leaflet-image-loaded {
         visibility: inherit;
 }
-.leaflet-control-layers fieldset {
-    border: none;
+
+/*
+ * Controls.
+ */
+
+.leaflet-bar a {
+  box-sizing: border-box;
+  width: 30px !important;
+  height: 30px !important;
+  line-height: 30px !important;
+  font-size: 24px !important;
+  font-weight: normal;
 }
-/* this is to indent the nested details in the layer control */
-/* details/summary is in progress 2018-03-02 on ms-edge so gonna rely on that...
-   https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/6261266-details-summary-elements */
-.leaflet-control-layers details.mapml-control-layers > details.mapml-control-layers {
-        margin-left: 1em;
+
+.leaflet-control-layers-toggle {
+  width: 36px !important;
+  height: 36px !important;
 }
-.leaflet-control-layers details.mapml-control-layers > * {
-        margin-left: 0;
-        margin-right: 0;
+
+.leaflet-bar a,
+.leaflet-control-layers {
+  border-color: #e3e3e3 !important;
 }
-.leaflet-control-layers details.mapml-control-layers > select,
-.leaflet-control-layers details.mapml-control-layers > span{
```

Comment:
This document:

```
<mapml>
    <head>
        <title>states</title>
        <base href="http://localhost:8080/geoserver/mapml" />
        <meta charset="utf-8" />
        <meta content="text/mapml;projection=OSMTILE" http-equiv="Content-Type" />
        <link href="http://localhost:8080/geoserver/mapml/topp:states/osmtile?style=pophatch" rel="style" title="pophatch" />
        <link href="http://localhost:8080/geoserver/mapml/topp:states/osmtile?style=polygon" rel="style" title="polygon" />
        <link href="http://localhost:8080/geoserver/mapml/topp:states/osmtile?style=" rel="self style" title="population" />
        <link href="http://localhost:8080/geoserver/mapml/topp:states/CBMTILE?style=" rel="alternate" projection="CBMTILE" />
        <link href="http://localhost:8080/geoserver/mapml/topp:states/APSTILE?style=" rel="alternate" projection="APSTILE" />
        <link href="http://localhost:8080/geoserver/mapml/topp:states/WGS84?style=" rel="alternate" projection="WGS84" />
    </head>
    <body>
        <extent units="OSMTILE">
            <input name="z" type="zoom" value="18" min="0" max="18" />
            <input name="txmin" type="location" rel="tile" position="top-left" axis="easting" units="tilematrix" min="-1.3885038382960921E7" max="-7455049.489182421" />
            <input name="tymin" type="location" rel="tile" position="bottom-left" axis="northing" units="tilematrix" min="2870337.130793682" max="6338174.0557576185" />
            <input name="txmax" type="location" rel="tile" position="top-right" axis="easting" units="tilematrix" min="-1.3885038382960921E7" max="-7455049.489182421" />
            <input name="tymax" type="location" rel="tile" position="top-left" axis="northing" units="tilematrix" min="2870337.130793682" max="6338174.0557576185" />
            <link tref="http://localhost:8080/geoserver/topp/wms?version=1.3.0&amp;service=WMS&amp;request=GetMap&amp;crs=urn:x-ogc:def:crs:EPSG:3857&amp;layers=states&amp;styles=&amp;bbox={txmin},{tymin},{txmax},{tymax}&amp;format=image/png&amp;transparent=true&amp;width=256&amp;height=256" rel="tile" />
            <input name="i" type="location" axis="i" units="tile" />
            <input name="j" type="location" axis="j" units="tile" />
            <link tref="http://localhost:8080/geoserver/topp/wms?version=1.3.0&amp;service=WMS&amp;request=GetFeatureInfo&amp;FEATURE_COUNT=50&amp;crs=urn:x-ogc:def:crs:EPSG:3857&amp;layers=states&amp;query_layers=states&amp;styles=&amp;bbox={txmin},{tymin},{txmax},{tymax}&amp;width=256&amp;height=256&amp;info_format=text/mapml&amp;transparent=true&amp;x={i}&amp;y={j}" rel="query" />
        </extent>
    </body>
</mapml>
```

rendered vertically in the previous version: 

![image](https://user-images.githubusercontent.com/2437285/92608473-69546e80-f283-11ea-8a41-ec947dd1fde2.png)

but now renders: horizontally.  Is that intended?

![image](https://user-images.githubusercontent.com/2437285/92608598-9143d200-f283-11ea-8cfb-00972f626591.png)

Reply:
That was not my intention, fixed in https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/164/commits/d4dc9a189053989717424b9f9da3451322259ac5:

![layers-list](https://user-images.githubusercontent.com/26493779/92644271-612c1b80-f2e3-11ea-918c-bc5dae1f1fa3.png)